### PR TITLE
Bogdannegura youtube modal

### PR DESCRIPTION
**Pull request by BogdanNegura**

### CODE REVIEW

**Comment by psyhedeliq** [comment]
Very nice @BogdanNegura !
:100:

File: src/_component/dialog/dialog.pug [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.dialog
+    .dialog-content
+        .dialog-header
+            span.close-dialog__icon.icon-cross
+        .dialog-body
+            <iframe id="video" width="560" height="270" src="" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (biancalina) [inline src/_component/dialog/dialog.pug]: I think you should write it with pug :D

File: src/_component/dialog/dialog.pug [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    function closeModal() {
+        dialog.style.display = "none";
+        video.src = ""
+    }
+
+    function clickOutside(e) {
+        if (e.target == modal) {
+            dialog.style.display = "none"
+        }
+    }
```
Comment (psyhedeliq) [inline src/_component/dialog/dialog.pug]: JavaScript :heart_eyes: :heart:

File: src/_component/dialog/dialog.pug [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    });
+    //- Get Close button
+    var closeButton = document.querySelector(".close-dialog__icon");
+
+    //- Events
+    //- dialogButton.addEventListener("click", openModal);
+    closeButton.addEventListener("click", closeModal);
+    //- window.addEventListener("click", clickOutside);
+
+    function openModal(event ) {
```
Comment (psyhedeliq) [inline src/_component/dialog/dialog.pug]: JavaScript :heart_eyes: :heart:

File: src/_component/dialog/dialog.pug (hunk 2 above)
Comment (BogdanNegura) [reply src/_component/dialog/dialog.pug]: Thank you Cornel but there are not all my merits, Marian shows me this way using #javaScript which was very useful in this case!

**Pull request merged**